app/code/Magento/CustomerGraphQl/etc/schema.graphqls

```diff
@@ -78,6 +80,34 @@ input CustomerInput {
     is_subscribed: Boolean @doc(description: "Indicates whether the customer is subscribed to the company's newsletter")
 }
 
+input CustomerCreateInput {
+    prefix: String @doc(description: "An honorific, such as Dr., Mr., or Mrs.")
+    firstname: String! @doc(description: "The customer's first name")
```

Comment:
can these be changed dynamically by a customization?

Reply:
What do you mean by customization? Making firstname non-mandatory field?

Reply:
I mean, the mandatory fields I think you can change them in the admin, or some extension can and this way is static..
since it was described in the proposal like this, we'll let it through..